infrastructure/lambda/template.yaml

```diff
@@ -63,6 +63,15 @@ Globals:
         POWERTOOLS_LOG_LEVEL: INFO
         SQS_AUDIT_EVENT_PREFIX: IPV_ADDRESS_CRI
 Mappings:
+
+  VPCEndpointIdMapping:
```

Comment:
assume these will be done via import eventually?

Reply:
yeah @mrdoodles  is exporting them in a VPC stack, but they're not available in all envs yet